docs/shared/mpas_streams.rst

```diff
@@ -0,0 +1,44 @@
+streams:
+^^^^^^^^
+
+See the "Configuring Model Input and Output" chapter in the *MPAS-Atmosphere Model User’s Guide*, available from the `MPAS releases page <https://mpas-dev.github.io/atmosphere/atmosphere_download>`_, for additional information, including the specifics of supported values, on these attributes..
+
+**Required attributes:**
+
+  ``filename_template:``
+
+  The template for files that exist or will be created by the stream. (type: string)
+
+  ``mutable:``
+
+  Whether the set of fields that belong to the stream may be modified at model run-time. (type: boolean)
+
+  ``name:``
+
+  A unique name used to refer to the stream. (type: string)
+
+  ``type:``
+
+  One of ``input``, ``output``, ``input;output``, or ``none``. (type: string)
+
+**Optional attributes:**
```

Comment:
The MPAS docs describe more optional attributes, similar to `packages` in
```
<stream name="foo", packages="bar;baz" />
```
(supported in this PR) and optional elements, similar to `file` in
```
<stream name="foo">
    <file name="bar"/>
</stream>
```
(supported in this PR via the `files:` item).

Since UW-556 calls for only implementing what's currently required, I implemented in this PR only what I saw used in the MPAS App examples I looked at. Please let me know if I missed something, or if you think we should implement more of the optional attributes/elements now rather than in follow-on work.